patches/common/chromium/color_chooser_mac.patch

```diff
@@ -0,0 +1,184 @@
+From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
+From: Benedek Heilig <benecene@gmail.com>
+Date: Thu, 21 Feb 2019 17:04:30 +0000
+Subject: Backports an upstream fix I made to the color chooser dialog on macos.
```

Comment:
this Subject becomes the commit message when the commits are applied, please keep it short & useful for `git log`!